### ISSUE TITLE
mapnik Filter : multibyte encoding in attribute name (e.g. utf-8 or sjis )

### ISSUE THREAD

**Issue by pockel** [mapnik/mapnik#1153]
I have Japanese geo-spatial data in shalefile format.
In this shapefile, some attribute's name have Shift-jis encoding characters.
I could retrieve the attribute name and the values in Shift-jis by osgeo.ogr Python binding.
So I tried to make an image map from this shapefile and use mapnik's Filter function.
Then I recieved error messages as followings,

```
 File "../P265_sjis.py", line 18, in <module>
    rule.filter = mapnik.Filter("[**] = ' --'")
RuntimeError: Failed to parse filter expression:
[**] = '--'
Parsing aborted at '['
```

(**, and -- are Japanese in Shift-JIS or UTF-8. )

Please support to treat multibyte encoding in Filter function.

The shapefiles are available from following links.
For example, the attribute name contains "名称", and "行政コード" in AdmArea shapefile.
http://www.pockel.net/shapefile/AdmArea.dbf    etc.
(These shapefile's content is of Shizuoka Prefecture of Japan, in 1/25000 scale.)
In addition of AdmArea, there are AdmBdry, AdmPt, BldA, Cntr, Cstline, ElvPt, RailCL, RdEdg, WL shapefiles.

**Comment by DeoLeung** [comment]
after 6 years I would like to ask the same question...

**Comment by DeoLeung** [comment]
I'm using the xml configue, using the xml declaration helps
```xml
<?xml version="1.0" encoding="utf-8"?>
```

**Comment by lightmare** [comment]
@DeoLeung yes, if you want utf-8 in your xml and it's not default on your system, you need to provide the encoding declaration.
Other than that, mapnik (master) has no problem having unicode attribute names, and using them in filters.
